src/test/java/com/icthh/xm/tmf/ms/customer/repository/timezone/DateTimeWrapper.java

```diff
@@ -0,0 +1,132 @@
+package com.icthh.xm.tmf.ms.customer.repository.timezone;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "jhi_date_time_wrapper")
+public class DateTimeWrapper implements Serializable {
```

Comment:
Why we need this wrapper?

Reply:
I think, this entity is added to jhipster tests to check that hibernate saves time in utc format.

Reply:
https://github.com/jhipster/generator-jhipster/pull/8501